Makefile

```diff
@@ -523,6 +536,9 @@ test: all
 	@./bin/pelz seal test/data/server_cert_test.der -o test/data/server_cert_test.der.nkl
 	@./bin/pelz seal test/data/client_priv_test.der -o test/data/client_priv_test.der.nkl
 	@echo "GEN => Test Key/Cert Files"
+	@cd kmyth/sgx/demo && make pre bin/ecdh-server bin/ecdh-client
+	@./kmyth/sgx/demo/bin/ecdh-server -r test/data/server_priv_test.pem -u test/data/client_cert_test.pem -p 7000 &
```

Comment:
I'd prefer to redirect the server output here to ```/dev/null```, I don't think seeing its logging adds anything useful.